VivistaServer/wwwroot/style.css

```diff
@@ -735,6 +735,40 @@ input.toggle + label:active::after {
 	visibility: hidden;
 }
 
+.chartBox {
+	width: 50%;
+	min-height: 500px;
+	min-width: 500px;
+	display: block;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.table-center {
+	margin-left: auto;
+	margin-right: auto;
+}
+
+	.table-center td{
+		text-align: center;
+		vertical-align: central;
+	}
+	.table-center th {
+		text-align: center;
+		vertical-align: central;
+	}
+

```

Comment:
Hier klopt iets niet met whitespace

Reply:
Dat klopt wel, omdat dit een verdere onderverdeling is van .table-center. Als ge steeds een laag dieper gaat, gaat er een tab komen.

Reply:
Zo doe ik het in de rest van de CSS ook niet, dus hier ermee beginnen is niet nodig.

Reply:
Ik heb het aangepast, maar als ik ctrl + k + d  doe, zet die automatisch zo'n tab erbij

Reply:
Irritant, is dat een VS of Resharper setting ergens?